examples/demo-rollup/tests/all_tests.rs

```diff
@@ -1,3 +1,6 @@
+// For now, exclude the evm from the bank tests
+#[cfg(not(feature = "experimental"))]
 mod bank;
+#[cfg(feature = "experimental")]
```

Comment:
Why do we need to exclude the evm here?

Reply:
There was some odd interaction between EVM and non-evm code which caused genesis to fail when EVM was not feature gated here. I'll investigate this afternoon.